plugins/search/providers/elastic_search/batch/KExportMediaEsearchEngine.php

```diff
@@ -10,10 +10,12 @@ class KExportMediaEsearchEngine extends KObjectExportEngine
 	const LIMIT = 10000;
 	
 	const PAGE_SIZE = 500;
+
+	const DATE_FORMAT = 'c';
 	
 	public function fillCsv(&$csvFile, &$data)
 	{
-		KalturaLog::info ('Exporting content for media items through Esearch');
+		KalturaLog::info ('Exporting content for media items through Esearch, data:'.json_encode($data));
```

Comment:
@TalDubovKaltura  why do you need to print the data here? It gets printed as part of the batch job object when the job starts running anyway.

Reply:
I just figured out at this point, when the process starting to fill the CSV that I don't have the info in the logs about the data that will be passed to the CSV generator. but if you think it's unuseful I can remove it